src/index.js

```diff
@@ -2,16 +2,17 @@ import React from 'react';
 import ReactDOM from 'react-dom';
 import './index.css';
 import App from './App';
+import Parent from './ParentComponent';
 import reportWebVitals from './reportWebVitals';
 
 ReactDOM.render(
+  <React.Fragment>
   <React.StrictMode>
     <App />
-  </React.StrictMode>,
+    <Parent/>
```

Comment:
<Parent /> should be inside <App /> component.
You should pass only 1 element in ReactDOM.render(**element**, container[, callback])

https://reactjs.org/docs/react-dom.html#reference

Reply:
Changes are done.